src/widgets-service.js

```diff
@@ -185,7 +194,8 @@ WidgetsService.prototype.createTimeSeriesModel = function (attrs, layer) {
 
   var widgetModel = new WidgetModel({
     id: attrs.id,
-    type: 'time-series'
+    type: 'time-series',
+    dataviewModelAttrsNames: ['column', 'columnType', 'bins', 'start', 'end']
```

Comment:
`column_type` or `columnType`? we have a problem with camelCase and underscore var names :S.